src/arch/ia32/address.rs

```diff
@@ -139,6 +144,12 @@ impl SubAssign for PhysicalAddress {
     }
 }
 
+impl Into<u64> for VirtualAddress {
```

Comment:
The intention is good but the documentation states that we should prefer implementing the `From` trait instead of implementing `Into` directly. It would be better to implement `From<VirtualAddress> for u64` in this case as `Into<u64>` for VirtualAddress would be implemented automatically.